### DISCUSSION TITLE
Anime Matrix Pictures

### DISCUSSION THREAD

**Discussion by yudapedia** [seerge/g-helper#272] [General]
Despite the bloats, Armoury Crate offers a lot of interesting Anime Matrix GIF to play with. 
I guess is it possible to "extract" GIFs from the Armoury Crate? I really like space-themed Anime Matrix,

**Comment by seerge** [comment]
@yudapedia it should be :) i don't have armoury installed atm, mb you can try to find analogs at https://giphy.com/?
